src/ert/async_utils.py

```diff
@@ -19,7 +19,7 @@ def new_event_loop() -> asyncio.AbstractEventLoop:
 
 def get_event_loop() -> asyncio.AbstractEventLoop:
     try:
-        return asyncio.get_event_loop()
+        return asyncio.get_running_loop()
```

Comment:
Not sure about this one. Based on the documentation: https://docs.python.org/3/library/asyncio-eventloop.html
`get_running_loop` can cause RuntimeError while get_event_loop will create one.

Reply:
It's fine then

Reply:
Yes, I assume that was the expected behavior.